docs/SD-JWT.md

```diff
@@ -0,0 +1,92 @@
+# SD-JWT Documentation
```

Comment:
we need the original license file of sd-jwt-python wiuthin this folder

and also an introduction that explains that this is a fork of sd-jwt-python using cryptojwt

Reply:
it seems that the License is the same as the one that we use on our own library. Do you want to add it in any case or just to link it in the readme with the suggested introduction and also do we want a separate readme or we need a separate one?